internal/output/release/release.go

```diff
@@ -205,6 +231,25 @@ func (o *Output) release(w io.Writer) error {
 	return nil
 }
 
+func (o *Output) releaseTemplate(filename string, w io.Writer) error {
+	_, err := os.Stat(filename)
+
+	if err != nil && errors.Is(err, os.ErrNotExist) {
+		if _, err := w.Write([]byte(output.Preamble("# "))); err != nil {
+			return err
+		}
+
+		tmpl, err := template.New("config").Parse(releaseTemplateStr)
+		if err != nil {
+			return err
+		}
+
+		return tmpl.Execute(w, o.meta)
+	}
+
+	return output.ErrSkip
```

Comment:
nit, but we could probably make code more readable by putting the skip condition on top:

```go
if err != nil {
   if errors.Is(err, os.ErrNotExist) { 
      return ErrSkip
   }
   
   return err
}

//... continue writing here
```

Reply:
updated, should be better now